doc/faq.rst

```diff
@@ -42,6 +42,25 @@ Installing and updating
       :alt: Navigator showing installing specific version of Spyder
 
 
+.. dropdown:: Q: Is commercial use allowed? What's the situation with Anaconda?
+
+   Spyder is 100% free and open source; there is no paid version or prohibition on commercial use.
+   Spyder is developed by its international user community, and supported by its users through `OpenCollective`_ and by its generous sponsoring organizations, including `Quansight`_ and `NumFOCUS`_.
+   Our source code, standalone installers and most of our distribution methods (Pip/PyPI, Linux distros, MacPorts, WinPython, etc) are fully unencumbered and can be freely redistributed, used and modified by anyone.
+
+   .. _OpenCollective: https://opencollective.com/spyder
+   .. _Quansight: https://www.quansight.com/
+   .. _NumFOCUS: https://numfocus.org/
+
+   The Anaconda distribution recently `recently changed`_ their EULA to add restrictions on large (> 200 employee) for-profit companies using it on a large scale, but that only applies to the package distribution infrastructure (the packaged Anaconda distribution and the ``defaults`` conda channel).
+   Instead, you can simply download the similar `Miniconda distribution`_, which is 100% open source and identical to full Anaconda except for the gigabytes of Python packages installed by default in the Anaconda ``base`` environment (which we recommend you avoid using anyway).
```

Comment:
```suggestion
   Instead, you can simply download the similar `Miniconda distribution`_, which is 100% open source and identical to full Anaconda except for the many Python packages installed by default in the Anaconda ``base`` environment (which we recommend you avoid using anyway).
```

- The phrasing seems derogatory, which reads opinionated and not something I think we should include in our FAQ.
- Also, perhaps we should point directly to Miniforge instead of Miniconda?

Reply:
> The phrasing seems derogatory, which reads opinionated and not something I think we should include in our FAQ.

Agreed

> Also, perhaps we should point directly to Miniforge instead of Miniconda?

Absolutely, didn't even know that existed. I'll do it when I pull and rebase these in, since I'll need to change the link target as well

Reply:
Done; also modified the follow sentences to account for that change.